ros2/ros2/Dockerfile

```diff
@@ -51,13 +55,16 @@ RUN apt-get update && apt-get install -q -y \
     && rm -rf /var/lib/apt/lists/*
 
 # setup environment
-RUN locale-gen en_US.UTF-8
+RUN apt-get update && apt-get install -y \
```

Comment:
For the ROS2 build, would just this work
```
ENV LANG C.UTF-8
ENV LC_ALL C.UTF-8
```
The same we know use in ros1 dockerfiles?